lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,2168 @@
+{
```

Comment:
<p>Tomu nerozumím. Proč je slicing aplikován na řetězec?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='108'/>

Reply:
<p>aby se nevypsal celý... je to prasárna?</p>